lua/notify/stages/util.lua

```diff
@@ -85,9 +85,9 @@ local function window_intervals(windows, direction, cmp)
 end
 
 function M.get_slot_range(direction)
-  local top = vim.opt.tabline:get() == "" and 0 or 1
+  local top = vim.opt.showtabline:get() == 0 and 0 or 1
   local bottom = vim.opt.lines:get()
```

Comment:
Since the MR title generally relates to improving the notification slot limit, I would like to add a proposal to consider incorporating the visible winbar.

```suggestion
  if vim.g.winbar == 1 then
    top = top + 1
  end
  local bottom = vim.opt.lines:get()
```

I'm not sure if this is the proper way to handle it. So I'll be glad for any suggestion for an improvement. 🙏